protocol-designer/src/pages/Designer/ProtocolSteps/StepForm/StepTools/MoveLiquidTools/LiquidClassesStepTools.tsx

```diff
@@ -18,10 +18,12 @@ import type { FieldPropsByName } from '../../types'
 interface LiquidClassesStepToolsProps {
   propsForFields: FieldPropsByName
   setShowFormErrors?: Dispatch<SetStateAction<boolean>>
+  mix?: boolean
```

Comment:
Let's go with a type here instead of a mix boolean
```suggestion
  type: 'transfer' | 'mix'
```